scripts/protocgen.sh

```diff
@@ -1,29 +1,37 @@
 #!/usr/bin/env bash
-
-set -eo pipefail
+set -euo pipefail
 
 : ${VERS:=master}
-URL_PATH=archive/
+URL_PATH=tarball/
 if [[ VERS -ne master ]]; then
-    URL_PATH=archive/refs/tags/v
+    URL_PATH=tarball/refs/tags/v
 fi
 
+echo "fetching proto files"
+
+# Get shortened ref of commit
+REF=`curl -H "Accept: application/vnd.github.v3.sha" -qL "https://api.github.com/repos/tendermint/spec/commits/${VERS}" | cut -c -7`
 # Edit this line to clone your branch, if you are modifying protobuf files
```

Comment:
I think maybe this is no longer true—if we want to swap the branch, the place to do it would be to edit `$VERS` above, right?